ec2/sc-portfolio-ec2-development.yaml

```diff
@@ -0,0 +1,152 @@
+      in this portfolio.
+    Default: ''
+  PrincipalGroupName1:
+    Type: String
+    Description: (Optional) Name of an IAM  group which can execute products
+      in this portfolio.
+    Default: ''
+  PrincipalRoleName2:
+    Type: String
+    Description: (Optional) Name of an IAM role which can execute products
+      in this portfolio.
+    Default: ''
+  PrincipalGroupName2:
+    Type: String
+    Description: (Optional) Name of an IAM  group which can execute products
+      in this portfolio.
+    Default: ''
+  RepoRootURL:
+    Type: String
+    Description: S3 root url for the repository containing the product templates.
+    Default: https://s3.amazonaws.com/aws-service-catalog-reference-architectures/
+  ProductVersionName:
+    Type: String
+    Description: The product version name in the service catalog
+Conditions:
+  CreateLaunchConstraint: !Equals
+    - !Ref 'LaunchRoleName'
+    - ''
+  CondPrincipalRoleName1: !Not
+    - !Equals
+      - !Ref 'PrincipalRoleName1'
+      - ''
+  CondPrincipalGroupName1: !Not
+    - !Equals
+      - !Ref 'PrincipalGroupName1'
+      - ''
+  CondPrincipalRoleName2: !Not
+    - !Equals
+      - !Ref 'PrincipalRoleName2'
+      - ''
+  CondPrincipalGroupName2: !Not
+    - !Equals
+      - !Ref 'PrincipalGroupName2'
+      - ''
+Resources:
+  SCEC2portfolio:
+    Type: AWS::ServiceCatalog::Portfolio
+    Properties:
+      ProviderName: !Ref 'PortfolioProvider'
+      Description: !Ref 'PorfolioDescription'
+      DisplayName: !Ref 'PorfolioName'
+  LaunchConstraintRole:
+    Type: AWS::CloudFormation::Stack
+    Condition: CreateLaunchConstraint
+    Properties:
+      TemplateURL: !Sub '${RepoRootURL}iam/sc-ec2vpc-launchrole.yaml'
+      TimeoutInMinutes: 5
+  LinkEndusersRole:
+    Type: AWS::ServiceCatalog::PortfolioPrincipalAssociation
+    Condition: CondPrincipalRoleName1
+    Properties:
+      PrincipalARN: !Sub 'arn:aws:iam::${AWS::AccountId}:role/${PrincipalRoleName1}'
+      PortfolioId: !Ref 'SCEC2portfolio'
+      PrincipalType: IAM
+  LinkEndusersGroup:
+    Type: AWS::ServiceCatalog::PortfolioPrincipalAssociation
+    Condition: CondPrincipalGroupName1
+    Properties:
+      PrincipalARN: !Sub 'arn:aws:iam::${AWS::AccountId}:group/${PrincipalGroupName1}'
+      PortfolioId: !Ref 'SCEC2portfolio'
+      PrincipalType: IAM
+  LinkEndusersAltRole:
+    Type: AWS::ServiceCatalog::PortfolioPrincipalAssociation
+    Condition: CondPrincipalRoleName2
+    Properties:
+      PrincipalARN: !Sub 'arn:aws:iam::${AWS::AccountId}:role/${PrincipalRoleName2}'
+      PortfolioId: !Ref 'SCEC2portfolio'
+      PrincipalType: IAM
+  LinkEndusersAltGroup:
+    Type: AWS::ServiceCatalog::PortfolioPrincipalAssociation
+    Condition: CondPrincipalGroupName2
+    Properties:
+      PrincipalARN: !Sub 'arn:aws:iam::${AWS::AccountId}:group/${PrincipalGroupName2}'
+      PortfolioId: !Ref 'SCEC2portfolio'
+      PrincipalType: IAM
+  ec2windowsjumpcloudproduct:
```

Comment:
should we remove this same product from sc-portfolio-ec2.yaml file?

Reply:
Yes, I can go ahead and do that on this PR. I do want to wrap this one up so that we have a pattern that I can apply to the S3 portfolios though.